### DISCUSSION TITLE
Pocketbase + Marmot = ❤️

### DISCUSSION THREAD

**Discussion by maxpert** [pocketbase/pocketbase#487] [Show and tell]
Hey folks,

 I've been working on a tool called [Marmot](https://github.com/maxpert/marmot), which helps you do multi-master replication among various nodes of SQLite. Few days ago I came across this lovely project, and it instantly clicked me that I can do a good demo with community, while I support and build an independent my OSS project. So I made a very short demo video for community here:

[![Scaling Pocketbase with Marmot](https://img.youtube.com/vi/VSa-VJso050/0.jpg)](https://www.youtube.com/watch?v=VSa-VJso050)

I would love to heard feedback, comments, or contributions to project from people using Pocketbase in production here. You can checkout [Marmot here](https://github.com/maxpert/marmot).

**Comment by benallfree** [comment]
@maxpert I added this to https://github.com/benallfree/awesome-pocketbase 👍  Thank you for this explanation!

**Comment by eqoram** [comment]
looks very promising! :)

**Comment by tigawanna** [comment]
> Thanks again. I did not mean at all btw to force nats as the solution. It’s just my own slant.
> 
> Your system looks really good and I hope to try it out.
> 
> mid youbfeel like it maybe update your docs addressing the things raised / answered in our discussion am with todobof what’s on the roadmap. 

Update from the future :The project did move to NATS 
 [ Marmot fly.io example ](https://github.com/maxpert/marmot-pocketbase-flyio) . This community is awesome
